dev_tools/packaging/verify-published-package.sh

```diff
@@ -61,13 +61,9 @@ cd "${tmp_dir}"
 trap "{ rm -rf ${tmp_dir}; }" EXIT
 
 # Test both the python 2 and python 3 versions.
```

Comment:
remove for loop?

Reply:
I assumed there are different python3 versions?

Reply:
That was the idea.